### PR TITLE
Movement labels gfx

### PR DESCRIPTION
**Pull request by trijeetm**

### CODE REVIEW

File: 2017/nomadic-hues/nameless-go.ck [hunk 1]
```diff
@@ -250,7 +250,7 @@ fun void client()
 {
 
   // the device number to open
-  0 => int deviceNum;
```
Comment (ntindall) [inline 2017/nomadic-hues/nameless-go.ck:253]: @trijeetm did you mean to change this?

File: 2017/nomadic-hues/world/world.pde [hunk 2]
```diff
@@ -35,6 +40,8 @@ void setup() {
   colorMode(HSB, 360, 100, 100, 100);
   frameRate(60);
 
+  _fontLabel = createFont("Calluna", 48, true);
```
Comment (ntindall) [inline 2017/nomadic-hues/world/world.pde:43]: hopefully it exists on the system we are running this on.

Comment (trijeetm) [reply 2017/nomadic-hues/world/world.pde:43]: yeah I just installed it

File: 2017/nomadic-hues/nameless-go.ck (hunk 1 above)
Comment (trijeetm) [reply 2017/nomadic-hues/nameless-go.ck:253]: crap no, the new macbook use this device numbers, I'll update the svn codebase to reflect the right device number

**Pull request merged**